connectors/survey.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import requests
+
+from models.comment import Comment
+
+class SurveyAPIConnector:
```

Comment:
Maybe rename SurveyConnector ? Just to keep consistency with the other connectors which also use APIs (git, Jira, etc)